publish.md

```diff
@@ -0,0 +1,7 @@
+### This file describes how to make a realse of the FHIR profiles and value sets. The release process is based on the auto-ig-builder
+
+1. Create an issue asking for a release, this should be approved by at least 2 organization members.
+2. When its been aprove create a tag in the format `vX.Y.Z` where X.Y.Z is the version number. This tag should be created in the main branch.
```

Comment:
Don't we need say that we are following Semantic Versioning 2.0.0 here? 
https://semver.org/

Reply:
@MangalaEkanayake perfect, I made an update to this